neuro_san/service/generic/chat_message_converter.py

```diff
@@ -73,3 +76,38 @@ def from_dict(self, obj_dict: Dict[str, object]) -> object:
                 to return None.
         """
         raise NotImplementedError
+
+    JSON_PRIMITIVES = (str, int, float, bool, type(None))
+    SKIP_TYPES = (ChatMessageType,)
+
+    def to_json_safe(self, obj: Any):
```

Comment:
Nice. Might be worthwhile to make a unit test for this and maybe eventually make its way down to leaf-common

Reply:
Curious though: What kinds of messages ended up being "offensive"?

I ask because in neuro_san/internals/messages, there is a notion of a TracedMessages, on which you can specify which fields should not be converted.  Maybe this ultimately belongs in that package somehow,
because that's where all the other ChatMessage-related stuff lives (This seems generic enough to not live in server-generic).